subprojects/environments/spark/spark-env.yml

```diff
@@ -0,0 +1,37 @@
+name: ipython-spark
+cluster:
+  docker:
+    type: $docker_instance_type
+    count: $docker_instance_count
```

Comment:
From the way the docker instance type is being used, it I believe will not work if $docker_instance_count is > 1. In any case, since the docker node is being used as the "master", it probably doesn't make sense to have anything more than 1? Probably best to hard code it to 1 to avoid confusion.